migration/i7.1z/oracle/202009111500_IDEMPIERE-3981.sql

```diff
@@ -0,0 +1,1036 @@
+;
+
+-- Jun 22, 2019, 3:53:59 PM BRT
+INSERT INTO AD_Field (AD_Field_ID,Name,Description,Help,AD_Tab_ID,AD_Column_ID,IsDisplayed,DisplayLength,SeqNo,SortNo,IsSameLine,IsHeading,IsFieldOnly,IsEncrypted,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsReadOnly,IsCentrallyMaintained,EntityType,AD_Field_UU,IsDisplayedGrid,SeqNoGrid,XPosition,ColumnSpan,NumLines,IsQuickEntry,IsDefaultFocus,IsAdvancedField) VALUES (206126,'DB Column Name','Name of the column in the database','The Column Name indicates the name of a column on a table as defined in the database.',200269,116,'Y',11,30,2,'Y','N','N','N',0,0,'Y',TO_DATE('2019-06-22 15:53:59','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2019-06-22 15:53:59','YYYY-MM-DD HH24:MI:SS'),100,'Y','Y','D','cd428dc7-129e-4d65-911d-e80acf0890e2','Y',30,4,2,1,'N','N','N')
+;
+
+-- Jun 22, 2019, 3:56:11 PM BRT
+INSERT INTO AD_Field (AD_Field_ID,Name,Description,Help,AD_Tab_ID,AD_Column_ID,IsDisplayed,DisplayLength,SeqNo,IsSameLine,IsHeading,IsFieldOnly,IsEncrypted,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsReadOnly,IsCentrallyMaintained,EntityType,AD_Field_UU,IsDisplayedGrid,SeqNoGrid,XPosition,ColumnSpan,NumLines,IsQuickEntry,IsDefaultFocus,IsAdvancedField) VALUES (206128,'Dynamic Validation (Lookup)','Override Dynamic Validation Rule for Lookup Window','For some situations the dynamic validation rule for a Lookup window should be different from user data entry window. ',773,213962,'Y',22,140,'Y','N','N','N',0,0,'Y',TO_DATE('2019-06-22 15:56:11','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2019-06-22 15:56:11','YYYY-MM-DD HH24:MI:SS'),100,'N','Y','D','c7b43c7f-0e01-499d-b127-19482aa9ffa7','Y',130,4,2,1,'N','N','N')
+;
+
+-- Jun 22, 2019, 3:57:01 PM BRT
+UPDATE AD_Field SET EntityType='D', AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_DATE('2019-06-22 15:57:01','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=206124
+;
+
+-- Jun 22, 2019, 3:57:10 PM BRT
+UPDATE AD_Field SET AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_DATE('2019-06-22 15:57:10','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=206126
+;
+
+-- Jun 22, 2019, 3:57:30 PM BRT
+UPDATE AD_Field SET SeqNo=360,IsDisplayed='Y', Updated=statement_timestamp(), UpdatedBy=100 WHERE AD_Field_ID=206126
```

Comment:
statement_timestamp() is not valid in oracle

Reply:
Fixed by using updated time of migration script generation